app/views/users/_form.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= f.input :name %>
+<%= f.input :email %>
```

Comment:
Usually we grab the whole form into the _form partial

Reply:
Done sha: 78a6e82